SDKPackageManager.ps1

```diff
@@ -0,0 +1,31 @@
+param([Parameter(Mandatory=$true)][string]$arch, [Parameter(Mandatory=$true)][string]$config, [Parameter(Mandatory=$true)][string]$task)
+
+$currentLocation = Get-Location
+$packagePath = "$currentLocation\BuildOutput\$config\$arch\"
+
+if(!$task.CompareTo("i"))
+{
+	write-host "installing packages"
+	add-appxpackage "$packagePath\Microsoft.WindowsAppRuntime.Framework\Microsoft.WindowsAppRuntime.Framework.msix"
+	add-appxpackage "$packagePath\DynamicDependency.DataStore.Msix\DynamicDependency.DataStore.msix"
+	add-appxpackage "$packagePath\DynamicDependencyLifetimeManager.Msix\DynamicDependencyLifetimeManager.msix"
+	add-appxpackage "$packagePath\PushNotificationsLongRunningTask.Msix\PushNotificationsLongRunningTask.msix"
+}
+elseif(!$task.CompareTo("u"))
+{
+	taskkill /IM PushNotificationsLongRunningTask.exe /F
+	taskkill /IM DynamicDependency.DataStore.exe /F
+	taskkill /IM DynamicDependencyLifetimeManager.exe /F
+
+	write-host "uninstalling packages"
+	remove-appxpackage "WindowsAppRuntime.Test.DDLM-4.1.1967.333-$($arch)_4.1.1967.333_$($arch)__8wekyb3d8bbwe"
```

Comment:
Is there a way to have a unit test that verifies the script runs successfully? The minute any package name is changed, the script will be broken.

Reply:
> Is there a way to have a unit test that verifies the script runs successfully?

I'm not following the use case.

UTs/FTs are written as TAEF tests. If they call ::Test::Bootstrap::Setup() that's canned code to correctly install the fake-WinAppSDK packages (WindowsAppRuntime.Test.DDLM... etc) and rigorously verify correctness else fail as expected.

Who's running this script? An interactive user for manual/monkey testing? Or TAEF UT/FT test code?

Reply:
Use case is a manual testing where I am installing all the test SDK packages manually. There are some E2E activation flows we want to test which require us to install all the packages. This is a tedious process when you are trying multiple iterations where remove-appxpackage has to be called since these packages have a dependency on the fwk package, so it's not easily replaceable. This script helps improve that process.